RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -2481,7 +2481,7 @@ public boolean removeRunCompression() {
   }
 
   // to be used with lazyor
```

Comment:
Making this function public is somewhat unfortunate. I guess it is fine. However, we will need to do a better job documenting them. This can be done after merging this PR.